test/e2e/framework.go

```diff
@@ -1685,6 +1735,20 @@ func (data *TestData) GetEncapMode() (config.TrafficEncapModeType, error) {
 	return config.TrafficEncapModeEncap, nil
 }
 
+func (data *TestData) IsProxyAll() (bool, error) {
```

Comment:
No need to make it public?

Reply:
Yes, no need to make it public. Changed it to private.